scripts/mkcloud

```diff
@@ -192,16 +192,19 @@ function error_exit
     exitcode=$1
     message=$2
     if [ -z "$SKIPSUPPORTCONFIG" ] ; then
+        local defaults="-l -I 5000"
+        local sc_cmd="supportconfig ${SUPPORTCONFIG_OPTIONS:-$defaults} | wc"
```

Comment:
pipes will not work in variables like this. better leave it in the old place.
The alternative would be to use 'eval' which will make pipes work